healthchecks.go

```diff
@@ -0,0 +1,45 @@
+package main
+import (
+	"net/http"
+	"fmt"
+	fthealth "github.com/Financial-Times/go-fthealth/v1a"
+)
+
+func (h Handlers) mapiCheck() fthealth.Check {
+	return fthealth.Check{
+		BusinessImpact:   "Articvle Preview Service will not work",
```

Comment:
'Article' not 'Articvle' - and I would say instead something like 'Editorial users won't be able to preview articles as they will appear on the website'